test/uk/gov/hmrc/helptosaveproxy/controllers/UCClaimantCheckControllerSpec.scala

```diff
@@ -24,20 +24,21 @@ import play.api.libs.json.Json
 import play.api.mvc.{Result ⇒ PlayResult}
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
-import uk.gov.hmrc.helptosaveproxy.TestSupport
+import uk.gov.hmrc.auth.core.retrieve.{Credentials, EmptyRetrieval, ~}
+import uk.gov.hmrc.helptosaveproxy.util.AuthSupport
 import uk.gov.hmrc.helptosaveproxy.connectors.DWPConnector
 import uk.gov.hmrc.helptosaveproxy.testutil.UCClaimantTestSupport
 import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class UCClaimantCheckControllerSpec extends TestSupport with UCClaimantTestSupport {
+class UCClaimantCheckControllerSpec extends AuthSupport with UCClaimantTestSupport {
 
   val mockDWPConnector = mock[DWPConnector]
 
-  val controller = new UCClaimantCheckController(mockDWPConnector)
+  val controller = new UCClaimantCheckController(mockDWPConnector, mockAuthConnector)
 
-  val nino = "WP010123A"
+  override val nino = "WP010123A"
```

Comment:
do we need this if we dont need to retireve nino ?

Reply:
I have removed the nino I added in AuthSupport as its not needed and then removed the override modifier here. So this code is not untouched by my changes.